### PR TITLE
Pivot

### PR DESCRIPTION
**Pull request by aebill14**
this was all ajax's doing, just I will take the blam id there is anything wrong :)

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/PivotSubsystem.java [hunk 1]
```diff
@@ -37,12 +37,13 @@ public PivotSubsystem(){
         m_PidController.setFF(kFF);
         m_PidController.setOutputRange(kMinOutput, kMaxOutput);
 
-        SmartDashboard.putNumber("pivotPOS", 73.08);
+        SmartDashboard.putNumber("pivotPOS", 5);
```
Comment (jc3m) [inline src/main/java/frc/robot/subsystems/PivotSubsystem.java:40]: I'm guessing you don't need this anymore?

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -118,6 +119,7 @@ public void autonomousPeriodic()
   @Override
   public void teleopInit()
   {
+    m_robotContainer.initialize();
```
Comment (jc3m) [inline src/main/java/frc/robot/Robot.java:122]: You probably do not want 2 of these `m_robotContainer.initialize();` commands. I would just delete this one and keep the one below the auto cancel

Comment (aebill14) [reply src/main/java/frc/robot/Robot.java:122]: I found it, thanks for pointing it out

File: src/main/java/frc/robot/subsystems/PivotSubsystem.java (hunk 1 above)
Comment (aebill14) [reply src/main/java/frc/robot/subsystems/PivotSubsystem.java:40]: I think it would still be good to have, just incase

**Pull request merged**